src/components/home/GetInvolved.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+
+import { spacing, Title2, Button } from '../styledComponents';
+import cardImage1 from '../../assets/homePage/CardImage_Volunteers.jpg';
+import cardImage2 from '../../assets/homePage/CardImage_SocialImpact.jpg';
+import cardImage3 from '../../assets/homePage/CardImage_Companies.jpg';
+
+const cards = [
+  {
+    image: { src: cardImage1, alt: 'Volunteers' },
+    title: 'Volunteers',
+    paragraph: 'Access a catalog of open-source projects and find opportunities that match your interests and skills.',
+  },
+  {
+    image: { src: cardImage2, alt: 'Social impact organizations' },
+    title: 'Social impact organizations',
+    // eslint-disable-next-line max-len
+    paragraph: 'As a social impact organization, you have the opportunity to showcase your Tech for Good projects and engage more qualified contributors.',
+  },
+  {
+    image: { src: cardImage3, alt: 'Companies' },
+    title: 'Companies',
+    // eslint-disable-next-line max-len
+    paragraph: 'Attract and meaningfully engage employees with Explore. Use our platform to build a purpose-driven culture and scale your skill-based volunteering program.',
+  },
+];
+
+const Container = styled(Grid)({ marginTop: spacing(6) });
+
+const Content = styled(Grid)({ marginLeft: spacing(10) });
+
+const CardsContainer = styled(Grid)({});
+
+const CardContainer = styled(Grid)({
+  height: '520px',
+  width: '330px',
+  padding: spacing(3),
+  '&:hover': {
```

Comment:
Let's try to standardize box shadows across the site too - make a "theme" box shadow (or a couple - maybe shadows `sm`, `md`, and `lg` (or just 2) or something like that?